packages/nbformat/src/index.ts

```diff
@@ -218,6 +218,11 @@ export interface IRawCellMetadata extends IBaseCellMetadata {
  * A raw cell.
  */
 export interface IRawCell extends IBaseCell {
+  /**
+   * A string field representing the identifier of this particular cell.
+   */
+  id: string;
```

Comment:
Can we add this field to the `IBaseCell`? Since it is the same for every type of cell, we don't need to specialize this field in each subinterface.

Reply:
See previous comment about `id` not being on the specification for unrecognized cells.